### PR TITLE
Turn on/off position requirement for rack

### PR DESCRIPTION
**Pull request by mkurek**
- added additional param to rack: require position
- when rack requires position, each asset belonging to this rack has to have non-empty position
- when rack does not require position (ex. warehouse-like rack), position could be empty (it's different than Ralph 2.0 behaviour, which required position for every asset)

### CODE REVIEW

**Comment by ar4s** [comment]
:ok:

File: src/ralph/data_center/models/physical.py [hunk 1]
```diff
@@ -375,12 +394,22 @@ def _validate_position_in_rack(self):
                 self.rack.max_u_height,
             )
             raise ValidationError({'position': [msg]})
+        if self.position is not None and self.position < 0:
+            msg = 'Position is lower than 0'
```
Comment (ghost) [inline src/ralph/data_center/models/physical.py]: `Position should be 0 or greater`?

File: src/ralph/data_center/models/physical.py [hunk 2]
```diff
@@ -176,6 +176,13 @@ class Rack(AdminAbsoluteUrlMixin, NamedMixin.NonUnique, models.Model):
         default=0,
     )
     accessories = models.ManyToManyField(Accessory, through='RackAccessory')
+    require_position = models.BooleanField(
+        default=True,
+        help_text=_(
+            'Uncheck if position is not required for this rack (ex. when rack '
```
Comment (ghost) [inline src/ralph/data_center/models/physical.py]: 'Uncheck if position is optional for this rack (ex. when rack '?
This is only a suggestion, do what you wish :)

**Comment by ghost** [comment]
:ok:

**Pull request merged**